src/components/Form.jsx

```diff
@@ -1,25 +1,56 @@
-const Form = () => (
-  <form className="flex flex-col gap-5 items-start justify-start">
-    <label htmlFor="title">
-      <input
-        type="text"
-        id="title"
-        placeholder="title"
-        className="border-2 border-black"
-      />
-    </label>
-    <label htmlFor="author">
-      <input
-        type="text"
-        id="author"
-        placeholder="author"
-        className="border-2 border-black "
-      />
-    </label>
-    <button type="submit" className="border-2 border-black px-2 py-1">
-      submit
-    </button>
-  </form>
-);
-
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { addBook } from "../redux/books/books";
+import store from "../redux/configureStore";
+function Form() {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState("");
+  const [author, setAutor] = useState("");
+  const handleSubmit = function (e) {
+    e.preventDefault();
+    if (!title && !author) return;
+    const book = {
+      id: new Date().getTime(),
```

Comment:
why not use a uuid or nanoid package instead... somethings more solid and professional..please check out [nanoid](https://www.npmjs.com/package/nanoid) or [UUID](https://www.npmjs.com/package/uuid)